src/App.js

```diff
@@ -1,10 +1,16 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
+import {Route} from "react-router-dom";
 
 const App = () => (
   <div>
+    
     <Navigation />
+    <Route exact path = "/" component ={""}/>
+    <Route path = "about" component ={""}/>
```

Comment:
Your about path is missing a forward slash '/'. The component should be equal to {About}